README.md

```diff
@@ -1,35 +1,35 @@
 ## What is it?
-Steam Buddy is a web-based tool for installing non-Steam software to your Linux based couch gaming system. It was primarily developed for GamerOS.
+Chimera is a web-based tool for installing non-Steam software to your Linux based couch gaming system. It was primarily developed for ChimeraOS.
 
 It also provides a set of tools for quality of life improvements for Steam.
 
 ## Installation
-On ArchLinux install `steam-buddy` package from the AUR. On GamerOS the package is pre-installed
+On ArchLinux install `chimera` package from the AUR. On ChimeraOS the package is pre-installed
 
-After installing the `steam-buddy` package, you must run the following commands to enable the web interface it and then restart your system:
+After installing the `chimera` package, you must run the following commands to enable the web interface it and then restart your system:
 ```
-    systemctl --user enable steam-buddy.service
-    sudo systemctl enable steam-buddy-proxy.service
-    sudo systemctl enable steam-buddy-proxy.socket
+    systemctl --user enable chimera.service
+    sudo systemctl enable chimera-proxy.service
+    sudo systemctl enable chimera-proxy.socket
 ```
 
 ## Usage
 
 ### Web interface
-You can connect to Steam Buddy on GamerOS by opening a browser on another computer and entering `gameros.local`. If that does not work, then determine the IP address of your GamerOS system by looking at the network settings and enter it directly into your browser.
+You can connect to Chimera on ChimeraOS by opening a browser on another computer and entering `gameros.local`. If that does not work, then determine the IP address of your ChimeraOS system by looking at the network settings and enter it directly into your browser.
```

Comment:
should be `chimeraos.local` instead of `gameros.local`

Reply:
Done